code/modules/admin/fun_verbs.dm

```diff
@@ -439,25 +371,40 @@
 
 /datum/admins/proc/sound_stop()
 	set category = "Fun"
-	set name = "Stop All Playing Sounds"
+	set name = "Stop Regular Sounds"
 
 	if(!check_rights(R_SOUND))
 		return
 
-	log_admin("[key_name(usr)] stopped all currently playing sounds.")
-	message_admins("[ADMIN_TPMONTY(usr)] stopped all currently playing sounds.")
 	for(var/mob/M in GLOB.player_list)
 		if(M.client)
 			SEND_SOUND(M, sound(null))
+
+	log_admin("[key_name(usr)] stopped regular sounds.")
+	message_admins("[ADMIN_TPMONTY(usr)] stopped regular sounds.")
+
+
+/datum/admins/proc/music_stop()
+	set category = "Fun"
+	set name = "Stop Playing Music"
+
+	if(!check_rights(R_SOUND))
+		return
+
+	for(var/mob/M in GLOB.player_list)
+		if(M.client)
 			var/client/C = M.client
 			if(C?.chatOutput && !C.chatOutput.broken && C.chatOutput.loaded)
```

Comment:
Since you added the `if(M.client)` check:
```suggestion
			if(C.chatOutput?.loaded && !C.chatOutput.broken)
```
Alternatively, to preserve the order of the checks turn these into an early continue instead.